src/MessagePane.tsx

```diff
@@ -25,15 +27,8 @@ class SuggestedActions extends React.Component<MessagePaneProps, {}> {
     }
 
     actionClick(e: React.MouseEvent<HTMLButtonElement>, cardAction: CardAction) {
-
-        //click is only valid if there are props.actions
```

Comment:
This was needed to "debounce" the handler, so that the button couldn't be clicked twice.